.changeset/vast-clubs-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/nextjs': patch
+---
+
+Adding ability to create a Clerk-compatible CSP header
```

Comment:
Let's rewrite this changeset to more clearly outline the new feature here. Write it such that someone reading the package release notes can clearly understand what's been added.